gridstatus/nyiso.py

```diff
@@ -441,8 +478,8 @@ def get_interconnection_queue(self, verbose=False):
             queue["Proposed COD"],
             errors="coerce",
         )
-        queue["Proposed  In-Service"] = pd.to_datetime(
-            queue["Proposed  In-Service"],
+        queue["Proposed In-Service"] = pd.to_datetime(
```

Comment:
should this be `"Proposed In-Service Date"` since that is the value in `active`?